nbdev_mkdocs/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.3.0"
```

Comment:
this is a new feature, please upgrade version to 0.4.0rc0

Reply:
Upgraded the version to 0.4.0rc0